app/src/main/java/com/duckduckgo/app/browser/BrowserActivity.kt

```diff
@@ -401,10 +416,13 @@ class BrowserActivity : DuckDuckGoActivity(), BookmarkDialogCreationListener, We
     @SuppressLint("SetJavaScriptEnabled")
     private fun configureWebView() {
         webView = layoutInflater.inflate(R.layout.include_duckduckgo_browser_webview, webViewContainer, true).findViewById(R.id.browserWebView) as WebView
+        defaultUserAgentString = webView.settings.userAgentString
+
         webView.webViewClient = webViewClient
         webView.webChromeClient = webChromeClient
 
         webView.settings.apply {
+            userAgentString = userAgentProvider.getUserAgent(defaultUserAgentString)
```

Comment:
Optional: Would it make sense to instantiate the userAgentProvider with the defaultUserAgentString?